code-challenges.js

```diff
@@ -17,26 +17,67 @@ const fruit3 = "cherry"
 const fruit4 = "kiwi"
 // Expected outcome: "cherry"
 
+// const stringSorter =(str1, str2) =>{
+//     if(str1.length > str2.length){
+//         return(fruit3)
+//     } else if(str1.length < str2.length){
+//         return(fruit4)
+//     } else ("same amount of characters")
+// }
+//     console.log(stringSorter(fruit3, fruit4))
+
 // Explain your code:
+//input is going to be our variables given which are fruit1-4
+//we want our code to output the string with the most amount of characters
+//create a function that will check how many characters are in each string 
+//function should also be able to compare 2 different strings 
+//code should return string with most characters  
+
+//i got the answer, however i know theres another more accurate way to achieve it. Was trying to use string interpolation but my code was missing something, after trouble shooting a bunch of times this was what i managed.   
```

Comment:
Your logic looks good!  You just have some syntax issues and missing a return, but otherwise great job!